Wrappers/Python/cil/framework/framework.py

```diff
@@ -3814,7 +3814,7 @@ def __init__(self,
         else:
             raise TypeError('array must be a CIL type DataContainer or numpy.ndarray got {}'.format(type(array)))
 
-        if array.shape != geometry.shape:
+        if numpy.squeeze(array).shape != geometry.shape:
             raise ValueError('Shape mismatch got {} expected {}'.format(array.shape, geometry.shape))
```

Comment:
Same here, is it possible for geometry.shape to be (1, X, Y)?

Reply:
No, the `ImageGeometry` and `AcquisitionGeometry`  have `shape` and `dimension_labels` properties that only report dimensions of length >1. Squeezing should always be ok in this respect, we could `reshape` too, but that is much looser as it could reshape the non-zero dimensions.

I've moved the `np.squeeze` to earlier in array checking.